smithy-aws-protocol-tests/model/rpcV2/empty-input-output.smithy

```diff
@@ -0,0 +1,173 @@
+$version: "2.0"
+
+namespace aws.protocoltests.rpcv2
+
+use smithy.protocols#rpcv2
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+
+@httpRequestTests([
+    {
+        id: "no_input",
+        protocol: rpcv2,
+        documentation: "Body is empty and no Content-Type header if no input",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
+        },
+        forbidHeaders: [
+            "Content-Type",
+            "X-Amz-Target"
+        ]
+        method: "POST",
+        uri: "/service/aws.protocoltests.rpcv2.RpcV2Protocol/operation/NoInputOutput",
+        body: ""
+    },
+    {
+        id: "no_input_server_allows_accept",
+        protocol: rpcv2,
+        documentation: "Servers should allow the Accept header to be set to the default content-type.",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Accept": "application/cbor",
```

Comment:
The server can't really honor sending a CBOR response here right? The operation has no output; the payload must be empty.